app/models/teacher.rb

```diff
@@ -1,6 +1,8 @@
 class Teacher < ApplicationRecord
   include DeclarativeUpdates
 
+  attr_accessor :confirm_failed_outcome
```

Comment:
Feels like if this is necessary then our model is doing more than it should. Might need help understanding why